docs/docs-content/clusters/edge/edge-host-management/configure-proxy.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: "Configure HTTP-Proxy in EMC"
+title: "Configure HTTP-Proxy in EMC"
+description: "Instructions for configuring HTTP proxy in Edge Management Console."
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge"]
+---
+
+You can configure your Edge hosts to use an HTTP-proxy for secure internet access from Edge Management Console(EMC).
```

Comment:
Let's explain why you would want to configure a proxy and when to configure. The answer is when you are in a restricted environment, such as most corporate networks, and internet access is controlled through a proxy server.

Reply:
The reason that I didn't explain this was it's still murky whether someone can use the EMC at all if they don't have a restricted environment so I wasn't sure what kind of wording to use and was planning on adding it after I have confirmed this with Engineering.